introduction/java_int_to_string/Solution.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Scanner;
+import java.security.*; //dont know what exactly is used, cant check properly
+
+public class Solution {
+  public static void main(String[] args) {
+    DoNotTerminate.forbidExit();
+    try {
+      Scanner input = new Scanner(System.in);
+      int inputNumber = input.nextInt();
+      input.close();
+      String stringFromNumber = new String(Integer.toString(inputNumber));
+      if (inputNumber == Integer.parseInt(stringFromNumber)) {
+        System.out.println("Good job");
```

Comment:
I guess this code was provided, but it's inconsistent, either put point in both case or in none of them.